docs/Fields.md

```diff
@@ -1106,6 +1106,132 @@ export const PostShow = (props) => (
 );
 ```
 
+## Translatable Fields
+
+You may have fields which are translated in multiple languages and want users to verify each translation. To display them, you can use the `<TranslatableFields>` components which expect the translatable fields to have the following structure:
+
+```js
+{
+    name: {
+        en: 'The english value',
+        fr: 'The french value',
+        tlh: 'The klingon value',
+    }
+}
+```
+
+This is how to use it:
+
+```jsx
+<TranslatableFields locales={['en', 'fr']}>
+    <TextField source="name">
+    <TextField source="description">
+</TranslatableFields>
+```
+
+You may override the locale selected by default using the `defaultLocale` prop which react-admin sets to the current UI locale when not provided.
```

Comment:
```suggestion
React-admin uses the user locale as the default locale in this field. You can override this setting by default using the `defaultLocale` prop:
```

Reply:
```suggestion
React-admin uses the user locale as the default locale in this field. You can override this setting using the `defaultLocale` prop:
```
?

Reply:
I find your sentence too long and harder to understand, which is why I suggested another one.

Reply:
I understood that, it's not obvious but I suggested to remove the second "default" in your suggestion